### PR TITLE
Inherit index.yaml from release-v2.6

### PR DESCRIPTION
**Pull request by aiyengar2**
Followup to https://github.com/rancher/charts/pull/1541. The index.yaml also needs to be pulled in.

---

This is the equivalent of running the following script:

```
git fetch upstream
git checkout upstream/release-v2.6 -- index.yaml
git add index.yaml
git commit -m "Inherit index.yaml from release-v2.6"
make charts
git add index.yaml
git commit --amend --no-edit
```

### CODE REVIEW

**Review by nickgerace — APPROVED** [review]

File: index.yaml [hunk 1]
```diff
@@ -976,16 +976,17 @@ entries:
       catalog.cattle.io/namespace: cattle-resources-system
       catalog.cattle.io/os: linux
       catalog.cattle.io/provides-gvr: resources.cattle.io.resourceset/v1
+      catalog.cattle.io/rancher-version: < 2.6.1-0
```
Comment (jiaqiluo) [inline index.yaml:979]: addition 1

File: index.yaml [hunk 2]
```diff
@@ -1256,15 +1257,16 @@ entries:
       catalog.cattle.io/namespace: cis-operator-system
       catalog.cattle.io/os: linux
       catalog.cattle.io/provides-gvr: cis.cattle.io.clusterscans/v1
+      catalog.cattle.io/rancher-version: < 2.6.1-0
```
Comment (jiaqiluo) [inline index.yaml:1260]: addition 2

File: index.yaml [hunk 3]
```diff
@@ -1675,15 +1673,16 @@ entries:
       catalog.cattle.io/namespace: cattle-gatekeeper-system
       catalog.cattle.io/os: linux
       catalog.cattle.io/provides-gvr: config.gatekeeper.sh.config/v1alpha1
+      catalog.cattle.io/rancher-version: < 2.6.1-0
```
Comment (jiaqiluo) [inline index.yaml:1676]: addition 3

**Review by jiaqiluo — APPROVED** [review]
LGTM

**Pull request merged**